src/templates/en/2019/index.html

```diff
@@ -25,8 +25,15 @@ <h2>
         Start exploring
       </a>
     </div>
-    <div class="intro-year">2019</div>
-    <img class="intro-image" src="/static/images/home-hero.png" alt="">
+
+    <svg class="intro-image" width="100%" height="100%" viewBox="0 0 715 812"
+    xmlns="http://www.w3.org/2000/svg" 
+    xmlns:xlink="http://www.w3.org/1999/xlink">
+      <text x="20" y="250" class="intro-year">2019</text>
+      <image xlink:href="/static/images/home-hero.png"
+      x="0" y="230"
+      height="562" width="715" />    
+    </svg>
```

Comment:
I haven't seen this technique before. What's the advantage of using an SVG wrapper for the image?

Reply:
I did this because I wanted to make the image and text responsive in the same width units.
Certainly, this is a bit tricky and has side effects.

Stop this hack, I think it is best to include letters in the illustrations.
Prepare new images with illustrations and 2019 text.
I fix it right away because.

Reply:
> I think it is best to include text in the illustrations

It might be easier to implement but I'm not sure about things like a11y and i18n. @obto WDYT?

Reply:
Just to clarify, we're talking about this "2019" text correct?
![screenshot-projects invisionapp com-2019 10 03-12_20_56](https://user-images.githubusercontent.com/684747/66149288-64e5f700-e5d8-11e9-845a-fda60880becd.png)

Assuming we are:
1. Text inside SVGs is accessible so there's no problem there, but I think it'd be easiest to put that inside the image itself, and just make sure it's readable when displayed at smaller sizes.
2. Is this graphic informative to readers? Or is it merely presentational? If it's the latter (which I feel it is) we can just give it `alt=""`

Reply:
I think @MSakamaki's most recent suggestion is to [put the text in the PNG itself](https://github.com/HTTPArchive/almanac.httparchive.org/blob/51650c655ed4ebdda90a8e26d3ed3462bd6f59b1/src/static/images/home-hero-2019.png). It's true that we can use alt to make it accessible, but I do feel like the "2019" text itself is important information. In the future when we have multiple editions of the Almanac, this will be the most prominent (only?) indicator of which version the user is currently viewing.

Another consideration is that the responsive mobile design separates the text from the graphic, so we need to have the ability to position them independently.

Reply:
Both versions can be accessible. For the images an alt of "The Web Almanac 2019 edition", or similarly a <title> inside the svg with the same text would work well.

So I think either option is good so long as its marked up correctly.

Reply:
Thanks @obto @rviscomi  for the review.

The reason I create the image is that prevent the 2019 word from becoming instability while loading the web fonts.

I agree "2019" text of important information.
Update alt attribute of image.

Reply:
For the mobile view we want to have the "2019" prominently at the top of the page, above the H1. Here's how it looks in the [design](https://projects.invisionapp.com/d/?origin=v7?origin=v7#/console/18174420/379965024/preview):

![image](https://user-images.githubusercontent.com/1120896/66246952-9351f800-e6e6-11e9-88b7-5fada9d8f9c8.png)

Here's how it appears in this branch currently: 

![image](https://user-images.githubusercontent.com/1120896/66246936-79181a00-e6e6-11e9-9f32-a8de342bf196.png)

Reply:
@rviscomi 
Sorry, I was looking at the old mobile layout.
I fixed the intro part to match the shared URL page design.